docs/sql-guide/functions/function-totimestamp.md

```diff
@@ -0,0 +1,78 @@
+---
+title: TOTIMESTAMP()
+layout: default
+parent: SQL functions
+grand_parent: SQL guide
+---
+
+# TOTIMESTAMP() function
+
+The `TOTIMESTAMP()` function converts the given integer value to a timestamp and returns it. It uses the [unix epoch](https://www.unixtutorial.org/unix-epoch/) as the base and calculates date-time equivalent of the integer.
+
+## Syntax
+
+```
+TOTIMESTAMP(int_exp, timeunit)
+```
+
+## Arguments
+
+
+| Argument | Description | Data type |
+|---|---|---|
+| `int_exp` | Required `int` value or expression to be converted to a `timestamp`. | `int` |
```

Comment:
So the user can enter a full expression or an integer value?

Can the DDL be altered to make that clearer?

e.g.,

```
TOTIMESTAMP([int_val | expr], timeunit)
```

Reply:
@ljsinclair a couple things:
1.  The majority of docs use str_expr or expr, so let's not add two inputs here. int_expr or expr is fine as either evaluate to an int. I'm going to cut a ticket for us to discuss standardizing
2. [] denotes optional, which arg 1 is not

Based on the above, we should have: 

`TOTIMESTAMP(int_expr, [timeunit])`